tasks/compose_Amazon.yml

```
@@ -0,0 +1,11 @@
+---
+# Setup for installing Docker compose v2 on Amazon from GitHub
+#
+# https://docs.docker.com/compose/install/linux/
+
+
+- name: Install Docker Compose V2 Plugin (Amazon)
+  ansible.builtin.get_url:
+    url: https://github.com/docker/compose/releases/latest/download/docker-compose-{{ ansible_system }}-{{ ansible_userspace_architecture }}
+    dest: /usr/libexec/docker/cli-plugins/docker-compose
+    mode: '0755'
```

We prefer to alphabetize where possible:
```suggestion
    dest: /usr/libexec/docker/cli-plugins/docker-compose
    mode: '0755'
    url: https://github.com/docker/compose/releases/latest/download/docker-compose-{{ ansible_system }}-{{ ansible_userspace_architecture }}
```